lib/experimental/Layouts/Utils/Tag/index.tsx

```diff
@@ -23,21 +25,27 @@ export const Tag = forwardRef<HTMLDivElement, TagProps>(
     return (
       <div
```

Comment:
shouldn't we make it a `button` or an `a`? this will help us dropping the `!!onClick` check in favor of `disabled`

Reply:
The thing is that if doesn't have an onClick action then it's not really a button or an a so then it would be semantically wrong.

Reply:
Then we should probably change the HTML element conditionally for accessibility purposes

Reply:
Okay, I see you're changing the `role` instead. It's probably ok then!